src/index.html

```diff
@@ -4,9 +4,318 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__content">
+        <a class="header__link" href="#index.html">
+          <img
+            src="/images/logo.png"
+            class="header__img"
+            alt="Moyo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple.html" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung.html" class="nav__link">Samsung</a>
+            </li>
```

Comment:
```suggestion
      <div class="header__content">
        <a class="header__link" href="#index.html">
          <img
            src="/images/logo.png"
            class="header__img"
            alt="Moyo"
          >
        </a>
        
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a href="#apple.html" class="nav__link is-active">Apple</a>
            </li>
            
            <li class="nav__item">
              <a href="#samsung.html" class="nav__link">Samsung</a>
            </li>
```
2'nd point in checklist